Dart/src/com/jetbrains/lang/dart/ide/errorTreeView/AnalysisServerStatusAction.java

```diff
@@ -0,0 +1,94 @@
+package com.jetbrains.lang.dart.ide.errorTreeView;
+
+import com.intellij.icons.AllIcons;
+import com.intellij.ide.actions.SendFeedbackAction;
+import com.intellij.openapi.actionSystem.AnActionEvent;
+import com.intellij.openapi.actionSystem.Presentation;
+import com.intellij.openapi.application.ApplicationManager;
+import com.intellij.openapi.extensions.Extensions;
+import com.intellij.openapi.project.Project;
+import com.intellij.openapi.ui.MessageDialogBuilder;
+import com.intellij.openapi.ui.Messages;
+import com.intellij.openapi.wm.ToolWindowManager;
+import com.jetbrains.lang.dart.DartBundle;
+import com.jetbrains.lang.dart.analyzer.DartAnalysisServerService;
+import org.jetbrains.annotations.Nullable;
+
+import javax.swing.*;
+
+public class AnalysisServerStatusAction extends SendFeedbackAction {
```

Comment:
Looks like this class doesn't use the fact that it extends `SendFeedbackAction`. All meaningful methods are overridden and have no `super()` calls. How about extending `DumbAwareAction` directly?